keyboards/mechmini/v2/keymaps/wsturgiss/keymap.c

```diff
@@ -0,0 +1,108 @@
+#include QMK_KEYBOARD_H
+#include "config.h"
+#include "../../../../../quantum/rgblight_list.h"
+
+#define base 0
+#define raise 1
+#define lower 2
+#define game 3
+#define _______ KC_TRNS 
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```